lib/node_modules/@stdlib/math/base/special/gammasgn/test/test.js

```diff
@@ -0,0 +1,156 @@
+var smallNegative = require( './fixtures/python/small_negative.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.equal( typeof gammasgn, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes the sign of the gamma function', function test( t ) {
+	var expected;
+	var x;
+	var v;
+	var i;
+
+	x = random.x;
+	expected = random.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		t.ok( v === expected[ i ], 'x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of the gamma function for small positive numbers', function test( t ) {
+	var expected;
+	var x;
+	var v;
+	var i;
+
+	x = smallPositive.x;
+	expected = smallPositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		t.ok( v === expected[ i ], 'x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of the gamma function for medium positive numbers', function test( t ) {
+	var expected;
+	var x;
+	var v;
+	var i;
+
+	x = mediumPositive.x;
+	expected = mediumPositive.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		t.ok( v === expected[ i ], 'x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of the gamma function for small negative numbers', function test( t ) {
+	var expected;
+	var x;
+	var v;
+	var i;
+
+	x = smallNegative.x;
+	expected = smallNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		t.ok( v === expected[ i ], 'x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function computes the sign of the gamma function for medium negative numbers', function test( t ) {
+	var expected;
+	var x;
+	var v;
+	var i;
+
+	x = mediumNegative.x;
+	expected = mediumNegative.expected;
+
+	for ( i = 0; i < x.length; i++ ) {
+		v = gammasgn( x[ i ] );
+		t.ok( v === expected[ i ], 'x: ' + x[ i ] + '. Value: ' + v + '. Expected: ' + expected[ i ] + '.' );
+	}
+	t.end();
+});
+
+tape( 'the function returns `NaN` if provided `NaN`', function test( t ) {
+	var v = gammasgn( NaN );
+	t.equal( isnan( v ), true, 'returns NaN' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided `+0`', function test( t ) {
+	var v = gammasgn( +0.0 );
+	t.equal( isPositiveZero( v ), true, 'returns 0' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided `-0`', function test( t ) {
+	var v = gammasgn( -0.0 );
+	t.equal( isPositiveZero( v ), true, 'returns 0' );
+	t.end();
+});
+
+tape( 'the function returns `+0` if provided a negative integer number', function test( t ) {
+	var end;
+	var v;
+	var i;
+
+	end = -100;
+
+	for ( i = 0; i >= end; i-- ) {
```

Comment:
```suggestion
	for ( i = 0; i >= -100; i-- ) {
```

The extra variable assignment is not necessary.